apps/src/templates/projects/PersonalProjectsTableActionsCell.jsx

```diff
@@ -70,17 +66,24 @@ export class PersonalProjectsTableActionsCell extends Component {
       <div>
         {!isEditing && (
           <QuickActionsCell>
-            <PopUpMenu.Item onClick={this.onRename}>
-              {i18n.rename()}
-            </PopUpMenu.Item>
+            {!this.props.isFrozen && (
+              <PopUpMenu.Item onClick={this.onRename}>
+                {i18n.rename()}
+              </PopUpMenu.Item>
+            )}
             <PopUpMenu.Item onClick={this.onRemix}>
               {i18n.remix()}
             </PopUpMenu.Item>
-            <MenuBreak />
-            <PopUpMenu.Item onClick={this.onDelete} color={color.red}>
-              <FontAwesome icon="times-circle" style={styles.xIcon} />
-              {i18n.delete()}
-            </PopUpMenu.Item>
+            {!this.props.isFrozen && <MenuBreak />}
+            {!this.props.isFrozen && (
```

Comment:
Documenting here that I initially had `MenuBreak` and `PopUpMenu.Item` wrapped in a div that required styling.
However, I got the following warning:
![warning](https://github.com/code-dot-org/code-dot-org/assets/107423305/8959bd3f-1016-48e9-800e-b6df3d405f1c)
So updated so that condition is checked for each component separately.